### PR TITLE
 #3936 issue resolved & Content update, improvement, Bug fix.

### LINKED ISSUE #3936: Feedback on page: /guides/v2.3/config-guide/redis/redis-session.html
# General issue

## Description:

Broken formatting in the topic.

## Possible solutions:

Fix formatting so it would be rendered with HTML appropriately.

### PR DESCRIPTION
**Pull request by hiteshvaghasiya83**
Fixes #3936.

## This PR is a:

- [ ] New topic
- [x] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

1. Add the link of system requirements for Magento (2.3).
2. Updated the **gdeurl** URL of Magento 2.0.
3. Add the link of PHP version for Magento (2.2, 2.3).

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/install-gde/install-flow-diagram.html
- https://devdocs.magento.com/guides/v2.3/install-gde/trouble/readiness/tshoot_rc_php.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3928) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3928) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@hiteshvaghasiya83 thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: _includes/install/trouble/rc_php-version.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ To resolve the issue, try the following:
 *	Use a [`phpinfo.php`]({{ page.baseurl }}/install-gde/prereq/optional.html#install-optional-phpinfo) page to collect more information
 *	Make sure you're running a supported PHP version according to our System Requirements:
 
-	*	[Version 2.0]({{ page.baseurl }}/install-gde/system-requirements.html)
+	*	[Version 2.0]({{ page.gdeurl }}/install-gde/system-requirements.html)
```
Comment (theatulsharma) [inline _includes/install/trouble/rc_php-version.md]: @hiteshvaghasiya83 
Please remove forward slash form below link.

**Old** 
> [Version 2.0]({{ page.gdeurl }}/install-gde/system-requirements.html)

**Upadated** 
> [Version 2.0]({{ page.gdeurl }}install-gde/system-requirements.html)

Comment (hiteshvaghasiya83) [reply _includes/install/trouble/rc_php-version.md]: @atulksigma 
Thanks for the suggestion. 
Suggested change is updated on rc_php-version.md file.

**Comment by hiteshvaghasiya83** [comment]
> ## This PR is a:
> * [ ]  New topic
> * [x]  Content update
> * [x]  Content fix or rewrite
> * [x]  Bug fix or improvement
> 
> ## Summary
> 1. Add the link of system requirements for Magento (2.3).
> 2. Updated the **gdeurl** URL of Magento 2.0.
> 3. Add the link of PHP version for Magento (2.2, 2.3).
> 
> ## Additional information
> List all affected URLs
> 
> * https://devdocs.magento.com/guides/v2.3/install-gde/install-flow-diagram.html
> * https://devdocs.magento.com/guides/v2.3/install-gde/trouble/readiness/tshoot_rc_php.html

**Comment by hiteshvaghasiya83** [comment]
> Hi @hiteshvaghasiya83, thank you for your contribution! 
> Please, complete [Contribution Survey](https://opensource.magento.com/survey/contribution/magento+devdocs+3928), it will take less than a minute. 
> Your feedback will help us to improve contribution process.

@magento-engcom-team, unfortunately, I have closed this pull request can you please reopen it.
I have resolved the issues of #3936.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/config-guide/redis/redis-session.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ functional_areas:
 
 ## Prerequisite {#reds-cache-prereq}
 
-Before you continue, [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install).
+If Redis will be used for default or page caching, you must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
```
Comment (hguthrie) [inline guides/v2.1/config-guide/redis/redis-session.md]:
```suggestion
You must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
```

File: guides/v2.1/config-guide/redis/redis-session.md [hunk 3]
```diff
@@ -54,9 +54,9 @@ where
...
-|session-save-redis-compression-threshold|compression_threshold|Set to 0 to disable compression (recommended when [suhosin.session.encrypt = On](http://suhosin.org/stories/howtos.html#encryption-features). Known issue with strings of more than 64KB.|2048|
+|session-save-redis-persistent-id|persistent_identifier|Unique string to enable persistent connections (for example, sess-db0).<br/>[Known issues with phpredis and php-fpm](https://github.com/nicolasff/phpredis/issues/70).
+|session-save-redis-db|database|Unique Redis database number, which is recommended to protect against data loss.<br/><br/><strong>Important</strong>: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
```
Comment (hguthrie) [inline guides/v2.1/config-guide/redis/redis-session.md]:
```suggestion
|session-save-redis-db|database|Unique Redis database number, which is recommended to protect against data loss.<br/><br/>**Important**: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
```

File: guides/v2.2/config-guide/redis/redis-session.md [hunk 4]
```diff
@@ -29,9 +33,9 @@ where
...
-|session-save-redis-compression-threshold|compression_threshold|Set to 0 to disable compression (recommended when [suhosin.session.encrypt = On](http://suhosin.org/stories/howtos.html#encryption-features). Known issue with strings of more than 64KB.|2048|
+|session-save-redis-persistent-id|persistent_identifier|Unique string to enable persistent connections (for example, sess-db0).<br/>[Known issues with phpredis and php-fpm](https://github.com/nicolasff/phpredis/issues/70).
+|session-save-redis-db|database|Unique Redis database number, which is recommended to protect against data loss.<br/><br/><strong>Important</strong>: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
```
Comment (hguthrie) [inline guides/v2.2/config-guide/redis/redis-session.md]:
```suggestion
|session-save-redis-db|database|Unique Redis database number, which is recommended to protect against data loss.<br/><br/>**Important**: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
```

File: guides/v2.2/config-guide/redis/redis-session.md [hunk 5]
```diff
@@ -85,13 +89,14 @@ Magento adds lines similar to the following to `<your Magento install dir>app/et
...
 
+{: .bs-callout .bs-callout-info }
 TTL for session records use the value for Cookie Lifetime, which is configured in Admin. If Cookie Lifetime is set to 0 (the default is 3600), then Redis sessions expire in the number of seconds specified in min_lifetime (the default is 60). This discrepancy is due to differences in how Redis and session cookies interpret a lifetime value of 0. If that behavior is not desired, increase the value of min_lifetime.
 
 ## Basic verification {#redis-verify}
 
 {% include config/redis-verify.md %}
 
-## Related topics
+#### Related topics
```
Comment (hguthrie) [inline guides/v2.2/config-guide/redis/redis-session.md]: H2 is consistent with other areas of the DevDocs collection. Revert this change.

File: guides/v2.2/config-guide/redis/redis-session.md [hunk 6]
```diff
@@ -7,7 +7,11 @@ functional_areas:
   - Setup
 ---
 
-You must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
+## Prerequisite {#reds-cache-prereq}
+
+If Redis will be used for default or page caching, you must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
```
Comment (hguthrie) [inline guides/v2.2/config-guide/redis/redis-session.md]:
```suggestion
You must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
```

File: guides/v2.2/config-guide/redis/redis-session.md [hunk 7]
```diff
@@ -7,7 +7,11 @@ functional_areas:
   - Setup
 ---
 
-You must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
+## Prerequisite {#reds-cache-prereq}
```
Comment (hguthrie) [inline guides/v2.2/config-guide/redis/redis-session.md]:
```suggestion
## Prerequisite
```

File: guides/v2.2/config-guide/redis/redis-session.md [hunk 8]
```diff
@@ -7,7 +7,11 @@ functional_areas:
...
+
+If Redis will be used for default or page caching, you must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
+
+## Configure Magento to use Redis for session storage {#config-redis-config}
```
Comment (hguthrie) [inline guides/v2.2/config-guide/redis/redis-session.md]:
```suggestion
## Configure Magento to use Redis for session storage
```

File: guides/v2.3/config-guide/redis/redis-session.md [hunk 9]
```diff
@@ -9,7 +9,7 @@ functional_areas:
 
 ## Prerequisite {#reds-cache-prereq}
 
-If Redis will be used for default or page caching, you must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing. .
+If Redis will be used for default or page caching, you must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
```
Comment (hguthrie) [inline guides/v2.3/config-guide/redis/redis-session.md]:
```suggestion
You must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
```

File: guides/v2.3/config-guide/redis/redis-session.md [hunk 10]
```diff
@@ -33,12 +33,9 @@ where
...
-Important: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
-|session-save-redis-compression-threshold|compression_threshold|Set to 0 to disable compression (recommended when [suhosin.session.encrypt = On](http://suhosin.org/stories/howtos.html#encryption-features).
-Known issue with strings of more than 64KB.|2048|
+|session-save-redis-persistent-id|persistent_identifier|Unique string to enable persistent connections (for example, sess-db0).<br/>[Known issues with phpredis and php-fpm](https://github.com/nicolasff/phpredis/issues/70).
+|session-save-redis-db|database|Unique Redis database number, which is recommended to protect against data loss.<br/><br/><strong>Important</strong>: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
```
Comment (hguthrie) [inline guides/v2.3/config-guide/redis/redis-session.md]:
```suggestion
|session-save-redis-db|database|Unique Redis database number, which is recommended to protect against data loss.<br/><br/>**Important**: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
```

File: guides/v2.0/config-guide/redis/redis-session.md [hunk 11]
```diff
@@ -13,7 +13,7 @@ functional_areas:
 
 ## Prerequisite   {#reds-cache-prereq}
 
-Before you continue, [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install).
+If Redis will be used for default or page caching, you must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
```
[the remark this answers is not in the document]
Comment (hiteshvaghasiya83) [reply guides/v2.0/config-guide/redis/redis-session.md]: I have removed 2.0 update changes.

File: guides/v2.2/config-guide/redis/redis-session.md (hunk 5 above)
Comment (hiteshvaghasiya83) [reply guides/v2.2/config-guide/redis/redis-session.md]: Done please check.

File: guides/v2.2/config-guide/redis/redis-session.md (hunk 6 above)
Comment (hiteshvaghasiya83) [reply guides/v2.2/config-guide/redis/redis-session.md]: Done requested changes please check.

File: guides/v2.2/config-guide/redis/redis-session.md (hunk 7 above)
Comment (hiteshvaghasiya83) [reply guides/v2.2/config-guide/redis/redis-session.md]: Done requested changes please check.

File: guides/v2.2/config-guide/redis/redis-session.md (hunk 8 above)
Comment (hiteshvaghasiya83) [reply guides/v2.2/config-guide/redis/redis-session.md]: Done requested changes please check.

File: guides/v2.3/config-guide/redis/redis-session.md (hunk 9 above)
Comment (hiteshvaghasiya83) [reply guides/v2.3/config-guide/redis/redis-session.md]: Done requested changes please check.

File: guides/v2.3/config-guide/redis/redis-session.md (hunk 10 above)
Comment (hiteshvaghasiya83) [reply guides/v2.3/config-guide/redis/redis-session.md]: Done requested changes please check.

**Comment by hguthrie** [comment]
running tests
